keyboards/dz60/keymaps/pinpox/rules.mk

```diff
@@ -0,0 +1,13 @@
+
+# Build Options
+# override the options specified in dz60/rules.mk
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
```
I recommend having at least Bootmagic lite enabled.

Reply:
> I recommend having at least Bootmagic lite enabled.

@Erovia I have noticed, that my keyboard does not work in the BIOS, which is a problem since I use the bootmanager (`f12`) to select the operating system to boot. Is there something I can do about that?

Also,  I cannot wake my computer from standby by pressing a key, which works with other keyboards. Any idea? Does that have to do with the BOOTMAGIC line?

Reply:
Depending on how old your BIOS is it may not recognise the NKRO interface. Try toggling it off (`NK_TOGG`).

Bootmagic Lite allows you to hold a certain key (conventionally the top left key) while powering the board on, which will make it jump to the bootloader for flashing. It shouldn't interfere with the BIOS at all.

Reply:
The `bootmagic` feature can change the "booting" of the keyboard. It has nothing to do with your computer's booting. :)

But as fauxpark mentioned, some BIOS does not handle NKRO. Turning it off usually helps.